src/test/datascience/notebook/helper.ts

```diff
@@ -116,7 +116,7 @@ export async function insertCodeCell(source: string, options?: { language?: stri
     }
     const startNumber = options?.index ?? activeEditor.document.cellCount;
     const edit = new WorkspaceEdit();
-    edit.replaceNotebookCells(activeEditor.document.uri, startNumber, 0, [
+    edit.replaceNotebookCells(activeEditor.document.uri, new NotebookRange(startNumber, 0), [
```

Comment:
@rebornix @IanMatthewHuff 
I'm not sure whether we need to merge this or not.
Looking at our main branch we have use `NotebookRange`.
However the arguments are different, in the main branch we are passing 
`edit.replaceNotebookCells(activeEditor.document.uri, new NotebookRange(startNumber, startNumber), [`

Where as @rebornix Pr is passing:

`edit.replaceNotebookCells(activeEditor.document.uri, new NotebookRange(startNumber, 0), [`

Which one is right?

Reply:
@DonJayamanne I rolled @rebornix 's changes into my big API merge. However some of the changes were wrong. The old API seemed to take 0 to mean, and this many further cells, but in the new API it means the end index only. I believe what I have in our code now is correct. But that's why you see a difference.